src/hooks/useMediaQuery/useMediaQuery.ts

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react';
+
+// Reference: src/styles/breakpoints.scss
+const breakpoints = {
+  lg: '1240px',
+  md: '905px',
+  sm: '600px',
+  xl: '1440px',
+} as const;
```

Comment:
Importar resoluções do SCSS se for possivel. Renomear as propriedades do objetos pra `from[res]`, tipo `from1240`, `from905` e etc